CoreFoundation/URL.subproj/CFURLSessionInterface.c

```diff
@@ -31,6 +31,10 @@ static CFURLSessionMultiCode MakeMultiCode(CURLMcode value) {
     return (CFURLSessionMultiCode) { value };
 }
 
+const char *CFURLSessionErrorDescription(int value) {
+    return curl_easy_strerror(value);
+}
```

Comment:
What's the memory management here? Does the result need to be freed?

Reply:
buffer is either static string or buffer set/owned by EasyHandle.swift. It doesn’t need to be freed.